config/sample-config.yaml

```diff
@@ -0,0 +1,806 @@
+server:
+  port: 9001
+  metricsPort: 9002
+loki: 
+  url: http://localhost:3100
+  labels: 
+    - SrcK8S_Namespace
+    - SrcK8S_OwnerName
+    - DstK8S_Namespace
+    - DstK8S_OwnerName
+    - FlowDirection
+    - Duplicate
+    - _RecordType
+  tenantID: netobserv
+  authCheck: none
+  useMocks: false
+frontend: 
+  recordTypes:
+    - flowLog
+  portNaming:
+    enable: true
+    portNames:
+      "3100": loki
+  columns: 
```

Comment:
For all the config below: do you remember if you changed some content or if it's only moving around these strings? (Just to help me reviewing, because I can't see the diff)